### DISCUSSION TITLE
DITA-OT 4.2

### DISCUSSION THREAD

**Discussion by infotexture** [dita-ot/dita-ot#4399] [Announcements]
### DITA-OT 4.2 has been released

✨  For details on what’s new, see the _Release Notes_ at [dita-ot.org/4.2/release-notes/](https://www.dita-ot.org/4.2/release-notes/).

⬇️ The [distribution package](https://github.com/dita-ot/dita-ot/releases/download/4.2/dita-ot-4.2.zip) is available from the [Downloads](https://www.dita-ot.org/download) page on the project website.

**Comment by stefan-jung** [comment]
Very interesting release. Thank you so much!
